### PR TITLE
Fix/truck inspection shorter keys

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
Update truck inspection client message format. 

Truck inspection client is publishing message with shorter keys in message params
## Related Issue
https://github.com/usdot-fhwa-stol/carma-platform/issues/877

https://github.com/usdot-fhwa-stol/carma-platform/issues/880
## Motivation and Context
ACE testing

## Types of changes
C++

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dan-du-car** [comment]
The current message format:
```
rostopic pub -r 10 mobility_operation_inbound cav_msgs/MobilityOperation "header: {sender_id: '', recipient_id: '', sender_bsm_id: '5053', plan_id: '', timestamp: 1586291248494}
strategy: "TruckInspection"
strategy_params: 'vin_number:<value>,license_plate:<value>,state_short_name:US'"

rostopic pub /mobility_operation_inbound cav_msgs/MobilityOperation "header: {sender_id: '', recipient_id: '', sender_bsm_id: '5055', plan_id: '', timestamp: 1586291827962}
strategy: 'TruckInspection'
strategy_params: 'vin_number:<value>,carrier_name:<value>,date_of_last_state_inspection:2020.01.20,date_of_last_ads_calibration:2020.02.20,license_plate:<value>,weight:<value>,carrier_id:1,permit_required:<value>,iss_score:<value>,pre_trip_ads_health_check:<value>,ads_health_status:5,ads_auto_status:<value>,ads_software_version:<value>,state_short_name:US'"
```
This is the new message format:
```
rostopic pub -r 10 mobility_operation_inbound cav_msgs/MobilityOperation "header: {sender_id: '', recipient_id: '', sender_bsm_id: '5053', plan_id: '', timestamp: 1586291248494}
strategy: "TruckInspection"
strategy_params: 'v:<value>,l:<value>,s:US'"

rostopic pub /mobility_operation_inbound cav_msgs/MobilityOperation "header: {sender_id: '', recipient_id: '', sender_bsm_id: '5055', plan_id: '', timestamp: 1586291827962}
strategy: 'TruckInspection'
strategy_params: 'v:<value>,n:<value>,f:2020.01.20,a:2020.02.20,l:<value>,w:<value>,c:1,p:Yes,i:75,r:<value>,h:1,u:engaged,o:<value>,s:US'"
```

File: truck_inspection_client/src/truck_inspection_client.cpp [hunk 1]
```diff
@@ -84,7 +84,8 @@ namespace truck_inspection_client
             mo_msg.strategy = this->INSPECTION_STRATEGY;
             std::string ads_auto_status = this->ads_engaged_ ? "Engaged" : "Not Engaged";
             std::string ads_health_status = ads_system_alert_type_ ;
-            std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            //std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            std::string params = boost::str(boost::format("v:%s,l:%s,n:%s,c:%s,w:%d,o:%s,f:%s,a:%s,r:%s,h:%s,u:%s,i:%d,p:%s")
```
Comment (maefromm) [inline truck_inspection_client/src/truck_inspection_client.cpp:88]: I thought we agreed to just do in alphabetical order? (e.g. a:%s,b:%s,c:%s, and so on). I think there's no meaning to the field names anymore that following first character doesn't say much on the other end. 

If you'd like to use what you have now, what letter logic did you choose from weight to iss_score (o:%s,f:%s,a:%s,r:%s,h:%s,u:%s,)?

File: truck_inspection_client/src/truck_inspection_client.cpp [hunk 2]
```diff
@@ -84,7 +84,8 @@ namespace truck_inspection_client
             mo_msg.strategy = this->INSPECTION_STRATEGY;
             std::string ads_auto_status = this->ads_engaged_ ? "Engaged" : "Not Engaged";
             std::string ads_health_status = ads_system_alert_type_ ;
-            std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
+            //std::string params = boost::str(boost::format("vin_number:%s,license_plate:%s,carrier_name:%s,carrier_id:%s,weight:%d,ads_software_version:%s,date_of_last_state_inspection:%s,date_of_last_ads_calibration:%s,pre_trip_ads_health_check:%s,ads_health_status:%s,ads_auto_status:%s,iss_score:%d,permit_required:%s")
```
Comment (maefromm) [inline truck_inspection_client/src/truck_inspection_client.cpp:87]: Please add a comment above that you've commented this out to put a temporary fix due to github issue#...

**Review by maefromm — CHANGES_REQUESTED** [review]

**Comment by maefromm** [comment]
This is PR goes hand in hand with https://github.com/usdot-fhwa-stol/carma-messenger/pull/48/files

**Review by maefromm — APPROVED** [review]

**Comment by maefromm** [comment]
@dan-du-car i have approved, but please look at my comment to add a comment in your code.

File: truck_inspection_client/src/truck_inspection_client.cpp (hunk 2 above)
Comment (dan-du-car) [reply truck_inspection_client/src/truck_inspection_client.cpp:88]: Added https://github.com/usdot-fhwa-stol/carma-platform/pull/882/commits/b01c1401bfb68bf4b2997afbed5adbff4b7d593e

**Review by maefromm — APPROVED** [review]

**Pull request merged**